modules/client-configuration/pages/registration-webui.adoc

```diff
@@ -14,6 +14,7 @@ By default, the SSH  port is [systemitem]``22``.
 By default, the username is [systemitem]``root``.
 . In the [guimenu]``Password`` field, type password to log in to the client.
 . In the [guimenu]``Activation Key`` field, select the activation key that is associated with the software channel you want to use to bootstrap the client.
+. In the [guimenu]``Proxy`` field, select the proxy from the available ones to register the system against it.
```

Comment:
```suggestion
. OPTIONAL: In the [guimenu]``Proxy`` field, select which proxy to register the client against.
```

Reply:
:question:

Reply:
yes, but ... I have the feeling that ,ost of the time we say "register to", "against" seems to occur 6 x.

Reply:
You're right. I'll edit.